src/schema/schema_drop.c

```diff
@@ -301,7 +301,7 @@ __drop_tiered(
      * released and closed. We have to know if the table is busy or if the close is successful
      * before cleaning up the tiered information.
      */
-    tiered_tmp = *tiered;
+    __wt_tsan_suppress_memcpy(&tiered_tmp, tiered, sizeof(tiered_tmp));
```

Comment:
I thought tiered_tmp is a pointer to tiered? Should we be doing an extra memcpy here?